### ISSUE TITLE
[Feature] Multi-cluster service mesh addon

### ISSUE THREAD

**Issue by shashankbarsin** [Azure/AKS#3266]

**Comment by SatyKrish** [comment]
Do you have plans to support Istio addon in AKS fleet? We are interested in multi-cluster load balancing, blue/green and failover scenarios using Istio/gateway API.

**Comment by ryanzhang-oss** [comment]
> Do you have plans to support Istio addon in AKS fleet? We are interested in multi-cluster load balancing, blue/green and failover scenarios using Istio/gateway API.

yes, we are collecting detailed use cases. Is your service deployed on multiple clusters?
